components/openzwave-adapter/src/lib.rs

```diff
@@ -352,6 +385,12 @@ impl OpenzwaveAdapter {
                         if kind.is_none() { continue }
                         let kind = kind.unwrap();
 
+                        let mut tags = HashSet::new();
+                        let label = vid.get_label();
+                        if !label.is_empty() {
+                            tags.insert(TaxoId::<TagId>::new(&vid.get_label()));
```

Comment:
This is a human-readable string that defines the service. I see this as a way for the user to know which device this is.